packages/app/src/specs/SpecsList.vue

```diff
@@ -259,6 +202,16 @@ const isProjectConnectOpen = ref(false)
 const isLoginOpen = ref(false)
 const loginUtmMedium = ref('')
 
+const cloudProjectType = computed(() => props.gql.currentProject?.cloudProject?.__typename)
```

Comment:
Same here - unless this is just for readability? 

I wonder if there's any perf overhead to `computed` when it only wraps a single value. We should find out.